feature-libs/product-configurator/rulebased/components/attribute/types/numeric-input-field/configurator-attribute-numeric-input-field.component.ts

```diff
@@ -32,9 +36,37 @@ export class ConfiguratorAttributeNumericInputFieldComponent
   @Input() language: string;
 
   @Output() inputChange = new EventEmitter<ConfigFormUpdateEvent>();
+  sub: any;
```

Comment:
sub: subscription;

Reply:
picked

Reply:
@ChristophHi, is it a breaking change to change a modifier from **public** to **protceted**?

Reply:
Yes, that would be breaking